x/market/client/cli/txDeal_test.go

```diff
@@ -1,26 +1,16 @@
 package cli
 
 import (
-	"github.com/medibloc/panacea-core/v2/types/testsuite"
-	"github.com/stretchr/testify/suite"
+	"github.com/stretchr/testify/require"
 	"testing"
 )
 
-type txTestSuite struct {
-	testsuite.TestSuite
-}
-
-func TestTxTestSuite(t *testing.T) {
-	suite.Run(t, new(txTestSuite))
-}
-
-// TestNewMsgCreateDeal
-// TODO: Test Client Command-Line MsgCreateDeal
-func (suite *txTestSuite) TestCmdCreateDeal() {
-
-}
+func TestParseSellDataFlags(t *testing.T) {
+	flags := CmdSellData().Flags()
+	err := flags.Set(DataVerificationCertificateFile, "./testdata/data_certificate_file.json")
 
-// TODO: Test Client Command-Line MsgSellData
-func (suite *txTestSuite) TestCmdSellData() {
+	require.NoError(t, err)
 
+	_, err = parseSellDataFlags(flags)
+	require.NoError(t, err)
```

Comment:
Why don't we assert that parsing results are the same to the expected values?

Reply:
Oh yes, I'll add it.